dashboard/app/views/scripts/show.html.haml

```diff
@@ -48,6 +48,8 @@
     -# Show all the levels, their names, and instructions in the extra links box.
     = render layout: 'shared/extra_links' do
       %strong= @script.name
+      %br/
+      = "id: #{@script.id}"
```

Comment:
This is here to make it easier for levelbuilders to find the script id. A future improvement could be to parse a copy&pasted level url, which would be easier for the levelbuilder; but, I think I want to cross that bridge only if we decide we actually want to invest more in this little tool

Reply:
![Screenshot 2025-01-22 at 11 16 09 AM](https://github.com/user-attachments/assets/8ed8d0b0-c1df-4bce-8736-7feb0dc75b11)

Reply:
I think it's a good idea, but don't want to block on it at the moment given that it's just me (and maybe Tess) as users.

Reply:
Makes sense! I was definitely thinking more of curriculum team members.